CMDDOC.md

```diff
@@ -455,3 +455,45 @@ When left unspecified, the default value for count is 1. Specifying a count valu
 #### Return value
 
 Array reply: list of popped elements and scores.
+
+### EXBZPOPMIN
+
+> EXBZPOPMIN key [key ...] timeout
+> time complexity: O(log(N)) with N being the number of elements in the sorted set.
+> Available for: Redis 6.0.0+
+
+`EXBZPOPMIN` is the blocking variant of the sorted set `EXZPOPMIN` primitive.
+
+It is the blocking version because it blocks the connection when there are no members to pop from any of the given sorted sets. A member with the lowest score is popped from first sorted set that is non-empty, with the given keys being checked in the order that they are given.
+
+The `timeout` argument is interpreted as a double value specifying the maximum number of seconds to block. A timeout of zero can be used to block indefinitely.
+
+See the [BLPOP documentation](https://redis.io/commands/blpop/) for the exact semantics, since `EXBZPOPMIN` is identical to `BLPOP` with the only difference being the data structure being popped from.
```

Comment:
```suggestion
See the [BLPOP documentation](https://redis.io/commands/blpop/) for the exact semantics, since `EXBZPOPMIN` is identical to `EXBLPOP` with the only difference being the data structure being popped from.
```

Reply:
This means the semantics is identical the original `BLPOP` command. And we don't have `EXBLPOP` in TairZset.

Reply:
OK